genai-perf/genai_perf/config/input/config_command.py

```diff
@@ -172,11 +174,32 @@ def _infer_subcommand(self) -> None:
     def _check_for_illegal_combinations(self) -> None:
         self._check_output_tokens_and_service_kind()
         self._check_output_format_and_generate_plots()
-        self._check_payload_input()
 
         self.endpoint.check_for_illegal_combinations()
         self.input.check_for_illegal_combinations()
 
+    def _check_fixed_schedule_requirements(self) -> None:
+        if self.perf_analyzer.stimulus == "fixed_schedule":
+            try:
+                with open(self.input.file, "r") as file:
+                    for line in file:
+                        self._check_for_timestamp_on_line(line)
+            except Exception as e:
+                raise ValueError(f"Error validating 'timestamp' field: {e}")
+
+    def _check_for_timestamp_on_line(self, line: str) -> None:
+        if not line.strip():
+            return
+        data = json.loads(line)
+        if "timestamp" not in data:
+            raise ValueError(
+                "User Config: Stimulus 'fixed_schedule' requires a 'timestamp' field in every entry in the input file."
+            )
+        if not isinstance(data["timestamp"], int):
+            raise ValueError(
+                "User Config: Stimulus 'fixed_schedule' requires the 'timestamp' field to be an integer."
+            )
+
```

Comment:
This checks that if a user uses `--fixed-schedule`, that the `--input-file payload:...` contains a `timestamp` integer key for every entry.